test/onnx/test_fx_to_onnx_with_onnxruntime.py

```diff
@@ -290,12 +293,19 @@ def _test_large_scale_exporter(
                 fake_args = create_args()
                 # Export ONNX model without initializers while ctx.paths records
                 # all files that contains real initializers.
-                (onnx_model, _, _, _) = fx_onnx.export_without_parameters_and_buffers(
-                    fake_model,
-                    *fake_args,
-                    use_binary_format=False,
-                    opset_version=self.opset_version,
-                    enable_dynamic_axes=enable_dynamic_axes,
+
+                onnx_model = (
```

Comment:
I suggest we can move these special test cases to another TestClass. It would be easier to maintain if anything changes in the future?

Reply:
Again, the intent of this PR is to plumb things through without breaking up existing tests. We will continue to iterate on this and parameterize tests around `ExportOptions`.